var/spack/repos/builtin/packages/root/package.py

```diff
@@ -835,14 +837,24 @@ def root_library_path(self):
             return "LD_LIBRARY_PATH"
         return "ROOT_LIBRARY_PATH"
 
+    def setup_root_env(
+        self,
+        env: spack.util.environment.EnvironmentModifications,
+        dependent: spack.package_base.PackageBase,
+    ):
+        """Add library directories from a dependent package to the environment."""
+        varname = self.root_library_path
+        for d in dependent.lib.directories:
+            env.prepend_path(varname, d)
```

Comment:
With your latest changes, is there a reason to keep the `setup_root_env` method?  I don't see it used in the PR.

Reply:
Nope, I meant to take that out. Thanks!